switchapi/es2k/lnw_v2/switch_pd_lag.c

```diff
@@ -17,15 +17,16 @@
 
 #include "switch_pd_lag.h"
 
-#include "switch_pd_p4_name_mapping.h"
-#include "switch_pd_utils.h"
+#include "switchapi/es2k/switch_pd_p4_name_mapping.h"
```

Comment:
This change is already being introduced in #120, so can be removed (or will result in conflict it that merges first)

Reply:
Merged without conflicts. 😊